packages/@expo/dom-webview/ios/DomWebViewModule.swift

```diff
@@ -0,0 +1,65 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+
+public final class DomWebViewModule: Module {
+  public func definition() -> ModuleDefinition {
+    Name("ExpoDomWebViewModule")
+
+    OnDestroy {
+      DomWebViewRegistry.shared.reset()
+    }
+
+    AsyncFunction("evalJsForWebViewAsync") { (webViewId: Int, source: String) in
+      if let webView = DomWebViewRegistry.shared.get(webViewId: webViewId) {
+        webView.injectJavaScript(source)
+      }
+    }
+
+    // swiftlint:disable closure_body_length
+    View(DomWebView.self) {
+      Events("onMessage")
+
+      Prop("source") { (view: DomWebView, source: DomWebViewSource) in
+        view.setSource(source)
+      }
+
+      Prop("injectedJavaScriptBeforeContentLoaded") { (view: DomWebView, script: String) in
+        view.setInjectedJSBeforeContentLoaded(script)
+      }
+
+      Prop("webviewDebuggingEnabled") { (view: DomWebView, enabled: Bool) in
+        view.webviewDebuggingEnabled = enabled
+      }
+
+      Prop("scrollEnabled") { (view: DomWebView, enabled: Bool) in
```

Comment:
The scroll view props are pretty nice, I'm also using the following in my prototype: 
```
contentInsetAdjustmentBehavior: "automatic",
automaticallyAdjustsScrollIndicatorInsets: true,
```

Would've also used a `scrollTo` to support scrolling to the top of a large header on tab, but it's not available in the RNWebView.